cpu/sam0_common/sam0_sdhc/sdhc.c

```diff
@@ -375,6 +375,7 @@ bool sdhc_send_cmd(sdhc_state_t *state, uint32_t cmd, uint32_t arg)
             if (--timeout == 0) {
                 SDHC_DEV->SRR.reg = SDHC_SRR_SWRSTCMD; /* reset command */
                 while (SDHC_DEV->SRR.bit.SWRSTCMD) {}
+                state->need_init = true;
```

Comment:
Are you really sure that the card needs to be reinitialized completely after `SWRSTCMD`? It only resets the CMD line, `SDHC_PSR_CMDINHC` and `SDHC_NISTR_CMDC`? If so, wouldn't it be required also here?https://github.com/RIOT-OS/RIOT/blob/a6a2071c642543c206ea5b3f4d49d13e8f989e25/cpu/sam0_common/sam0_sdhc/sdhc.c#L369